code-of-conduct/index.html

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+---
+<section class="articles pdt-50 pdb-50">
+  <div class="columns">
+    <div class="column is-8">
+      <div class="card article">
+        <div class="card-content">
+          <div class="media">
+            <div class="media-content has-text-centered">
+              <p class="title article-title">Ruby Nepals Code of Conduct</p>
+              <p class="subtitle is-6 article-subtitle">
+		Version: 0.1 (2022-09-06)
+              </p>
+            </div>
+          </div>
+          <div class="content article-body">	    
+	    <h3>Matz is nice, and we are nice.</h3>
+	    <p>To report an incident, please review the reporting guide and email conduct@rubynepal.org.</p>
```

Comment:
Suggestion to add href for the email along with basic template.

```suggestion
	    <p>To report an incident, please review the reporting guide and email <a href="mailto:conduct@rubynepal.org?cc=rubynepal.org@gmail.com&subject=Reporting Violation of Code of Conduct&body=This incident violates CoC of RubyNepal which happened to me or I witnessed or the victim notified me that ...">Email Us</a>.</p>
```